### ISSUE TITLE
Page boundaries can break lines apart

### ISSUE THREAD

**Issue by vinczemarton** [mrafiqk/html-pdf-node#51]
I'm having a meltdown over this one:

<img width="652" alt="image" src="https://user-images.githubusercontent.com/37654573/145439646-25fdc74d-596b-4f8c-97a0-b78c8f8419ef.png">

It does not show up on very simple documents. Page breaks work properly most of the time so I was thinking something like using tables as layout could break this but then I remade it into simple divs and the problem is still present.

The text length is entirely user data so I cannot control it.

**Comment by Nikola-Milovic** [comment]
Have you resolved this issue somehow? @vinczemarton

**Comment by vinczemarton** [comment]
No, we made the design more concise so there is less chance that we are generating multiple pages. I'm still balding because of this though.

**Comment by Nikola-Milovic** [comment]
@vinczemarton check out [html-pdf](https://www.npmjs.com/package/html-pdf), does the same thing with practically the same code, it worked for me while html-pdf-node didn't. (I had issues with puppeteer on alpine)

**Comment by vinczemarton** [comment]
Unfortunately html-pdf uses phantomjs with awful rendering issues (like weird dpi, bold charaters not working out of the box, flexbox not working). We might go for it still but it is just painful
